test/backend/info/technicianDetails.test.ts

```diff
@@ -0,0 +1,27 @@
+import { TechnicianDetailsBuilder, TechnicianType } from "../../../lib/backend/info/technicianDetails";
+
+describe("TechnicianDetails", () => {
+    it("should have a valid constructor", () => {
+        const TechnicianDetails = new TechnicianDetailsBuilder()
+            .withWorktype(TechnicianType.ELECTRICIAN)
+            .withResponseTime(500)
+            .withHourlyRate(500)
+            .build();
+        expect(TechnicianDetails.worktype).toBe("Electrician");
+        expect(TechnicianDetails.responseTime).toBe(500);
+        expect(TechnicianDetails.hourlyRate).toBe(500);
+
+    })
+
+    it("should have a valid constructor", () => {
```

Comment:
Same message is confusing, what are you trying to achieve here? instead it would be better to add dest for default constructor. e.g. const technicianDetail = TechnicianDetailsBuilder.build()